crates/driver/src/infra/config/file/mod.rs

```diff
@@ -260,6 +260,9 @@ struct TenderlyConfig {
 struct EnsoConfig {
     /// URL at which the trade simulator is hosted
     url: Url,
+    /// How often the network produces a new block. If this is not set the
+    /// system assumes an unpredictable network like proof-of-work.
+    network_block_interval_ms: Option<u64>,
```

Comment:
very optimistic to make this configurable in milliseconds.

Reply:
Just so you can set half seconds. Probably the actual way to go would be to use the human readable time parser Ilya recently discovered.